spec/models/recipe_spec.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Recipe, type: :model do
```

Comment:
Because you have [this line](https://github.com/orhanna14/homebrew-buddy/blob/d99450f/spec/rails_helper.rb#L51) in your rails_helper, you don't need `type: :model` it will figure it out automatically based on the fact that it's in the `models` directory. Feel free to remove it, or not! It's also not hurting anything 🙂

Reply:
Right! I saw that line in use in a project yesterday. Thanks for pointing that out.